Sources/Client/DeltaClientApp.swift

```diff
@@ -127,9 +130,17 @@ struct DeltaClientApp: App {
     Self.modalState.update(to: .error(message, safeState: safeState))
   }
   
-  /// Logs a fatal error and then fatal errors.
-  static func fatal(_ message: String) -> Never {
+  /// Logs a fatal error and redirects to troubleshooting page.
+  static func fatal(_ message: String) {
```

Comment:
With this function, I think it's best to show the fatal error message as a permanent element of the fatal error page instead of as a modal popup, because users will want to be able to take their time reading the fatal error

Reply:
Plain text on top of the buttons?

Reply:
yeah i reckon they should just be sent to a screen with the error as a Text and troubleshooting buttons underneath